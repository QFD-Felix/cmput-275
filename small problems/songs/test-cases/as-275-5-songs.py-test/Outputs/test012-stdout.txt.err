11 8 2 22 1 3 0 15 2 0 17 1 2 3 10 0 1 4 0 7 0 0 16 0 7 5 7 1 0 0 4 1 3 6 8 1 1 0 0 1 0 0 0 0
