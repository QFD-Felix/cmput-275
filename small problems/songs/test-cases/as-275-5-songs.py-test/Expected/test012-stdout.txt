11 8 2 22 1 3 0 15 2 0 17 1 2 3 10 0 1 4 0 7 0 0 17 0 7 5 7 1 0 0 5 1 4 1 9 5 1 1 0 0 1 0 0 0 0
